### ISSUE TITLE
MinDCF result

### ISSUE THREAD

**Issue by RMobina** [TaoRuijie/ECAPA-TDNN#80]
In the paper, the minDCF results are reported with P_target=0.01, but in the source code is set to 0.05. 

`minDCF, _ = ComputeMinDcf(fnrs, fprs, thresholds, 0.05, 1, 1)
`
I would appreciate it if you could clarify whether the results are based on a P_target of 0.01 or 0.05.

also, I have developed different extended versions of ECAPA-TDNN model with  p_target = 0.05, and I want to evaluate how it performs with a p_target = 0.01, should I simply adjust the p_target = 0.01 and evaluate my pretrained version with `python trainECAPAModel.py --eval --initial_model exps/pretrain.model` or if is necessary the model trains again with p_target = 0.01?

Thank you.

**Comment by TaoRuijie** [comment]
p_target is only related to minDCF, I use EER as the main metric so I did not care it too much.

Yes, for the trained model, just adjust P_target during evaluation is ok.

**Comment by RMobina** [comment]
Thank you for your clarification.
